src/sagemaker/image_uri_config/pytorch.json

```diff
@@ -102,6 +102,7 @@
                     "ap-east-1": "057415533634",
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
```

Comment:
I don't see KIX under the EIA section here, was that intended?

Reply:
@saimidu can you confirm if it's needed or not here?

Reply:
EIA images are only supported in a subset of AWS regions.

Reference:
- https://github.com/aws/deep-learning-containers/blob/master/available_images.md#available-deep-learning-containers-images
- https://aws.amazon.com/machine-learning/elastic-inference/pricing/

ap-northeast-3 is not one of these regions, and therefore does not need to be included.